vscode-lean4/src/infoview.ts

```diff
@@ -136,6 +136,7 @@ export class InfoProvider implements Disposable {
                 if (this.webviewPanel && this.client.initializeResult && window.activeTextEditor){
                     await this.webviewPanel.api.initialize(this.client.initializeResult, this.getLocation(window.activeTextEditor));
                 }
+                void this.webviewPanel?.api?.serverRestarted()
```

Comment:
Why isn't `webviewPanel.api.initialize` enough?  With this branch I'm seeing the inital startup case is broken where infoview starts working then this even comes in and effectively kills it, leaving the infoview in a non-working state.

Reply:
This is what I mean by infoview not working, notice the numbers are not updating as I edit the .lean file:

https://user-images.githubusercontent.com/18707114/139720209-a52e5617-1ad8-4fda-b94b-eb44db11abe9.mp4

Reply:
Ah, this is probably a race condition.  It works just fine here on my machine.

Reply:
Infoview now works with your latest changes, so I'm resolving this conversation.